src/main/kotlin/com/redhat/devtools/intellij/kubernetes/model/resource/NamespacedResourceOperator.kt

```diff
@@ -68,16 +66,18 @@ abstract class NamespacedResourceOperator<R : HasMetadata, C: Client>(
         }
         @Suppress("UNCHECKED_CAST")
         val typedWatcher = watcher as? Watcher<R> ?: return null
+        val inNamespace = this.namespace ?: return null
```

Comment:
At the beginning of this method there is already a check for `namespace == null`